src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -21,17 +21,17 @@ pub struct TargetInfo {
 pub enum FileFlavor {
     /// Not a special file type.
     Normal,
-    /// It is something you can link against (e.g. a library)
+    /// Something you can link against (e.g., a library).
     Linkable,
-    /// It is a piece of external debug information (e.g. *.dSYM and *.pdb)
+    /// Piece of external debug information (e.g., `.dSYM`/`.pdb` file).
     DebugInfo,
 }
 
 pub struct FileType {
     pub flavor: FileFlavor,
     suffix: String,
     prefix: String,
-    // wasm bin target will generate two files in deps such as
+    // Wasm bin target will generate two files in deps such as
```

Comment:
If we're going to capitalise, should this be "WASM"?

Reply:
The official capitalisation is actually "Wasm". I know, surprised me too.